### PR TITLE
adjusted index node name to include a prefix.

### LINKED ISSUE #769: Adjust index node name (again)
The guidelines have been clarified meanwhile in https://experienceleague.adobe.com/en/docs/experience-manager-learn/foundation/development/understand-indexing-best-practices#fully-custom-index on how a fully custom index name should look like:

> `<prefix>.<customIndexName>-<version>-custom-<customVersion>`

Our index name does not comply with that: https://github.com/Netcentric/accesscontroltool/blob/develop/accesscontroltool-oakindex-package/src/main/jcr_root/_oak_index/repACL-custom-1.xml

### PR DESCRIPTION
**Pull request by ofarganc**

### CODE REVIEW

**Review by ghenzlercn — APPROVED** [review]

File: accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml [hunk 1]
```diff
@@ -14,5 +14,5 @@
 
 <workspaceFilter version="1.0">
     <filter root="/oak:index/repACL" type="cleanup"/><!-- legacy index node name, used prior version 2.7.2  -->
-    <filter root="/oak:index/repACL-custom-1"/>
```
Comment (kwin) [inline accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml:17]: we need to remove the old one, please keep the old filter rule with type="cleanup"

File: accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml [hunk 2]
```diff
@@ -14,5 +14,6 @@
...
+    <filter root="/oak:index/repACL-custom-1" type="cleanup"/>
+    <filter root="/oak:index/actool.repACL-custom-1"/>
```
Comment (kwin) [inline accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml]: This name does not comply with `<prefix>.<customIndexName>-<version>-custom-<customVersion>` either (https://experienceleague.adobe.com/en/docs/experience-manager-learn/foundation/development/understand-indexing-best-practices#fully-custom-index)

File: accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml [hunk 3]
```diff
@@ -14,6 +14,6 @@
 
 <workspaceFilter version="1.0">
     <filter root="/oak:index/repACL" type="cleanup"/><!-- legacy index node name, used prior version 2.7.2  -->
-    <filter root="/oak:index/repACL-custom-1" type="cleanup"/>
+    <filter root="/oak:index/repACL-1-custom-1" type="cleanup"/>
```
Comment (kwin) [inline accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml]: You adjusted the wrong filter root (the old one to be removed)! Therefore also the build fails.

Comment (ofarganc) [reply accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml]: sorry 😓  fixed.

**Pull request merged**